app/src/main/java/com/nmakademija/nmaakademija/entity/Article.java

```diff
@@ -1,8 +1,9 @@
 package com.nmakademija.nmaakademija.entity;
 
-import java.io.Serializable;
+import android.os.Parcel;
+import android.os.Parcelable;
 
-public class Article implements Serializable {
+public class Article implements Parcelable {
```

Comment:
One more reason to use `Parcelable` here is because we don't have method `getSerializableArrayList`only `getParcelableArrayList`